src/scss/_variables.scss

```diff
@@ -1,3 +1,4 @@
 $header-font-color: #5C75DA;
-$body-background-color: #EAEBEF;
 $data-card-background-color: #FEFEFE;
+$body-background-color: #322C65;
+$yellow-data-info: #F2BC33;
```

Comment:
After I merged both our branches locally I ran another accessibility audit and this yellow color got flagged for not enough contrast - but seemingly only in some of the places it was used.  I think we should still merge but I've added it to the accessibility issue we already have open.